examples/simple-vpc/main.tf

```diff
@@ -9,7 +9,7 @@ module "vpc" {
 
   cidr = "10.0.0.0/16"
 
-  azs             = ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
+  zones           = ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
```

Comment:
(minor) I would probably even name it `availability_zones`

Reply:
Yes, that makes it more explicit but will require some testing.